sections/introduction.include

```diff
@@ -1,16 +1,21 @@
 <section>
+
 <!--
 
-This is the source file for Introduction.html
-All the content is "non-normative". It contains
-- a background / potted history of HTML
-- a few design notes
-- a note on HTML and XML syntax
-- information on how the spec is written and how to read it
-- some privacy considerations
-- an introduction to HTML itself, how to write it, etc.
-- non-normative conformance requirements for content
-- links to some other specs: charmod, unicode security, and some accessibility specs
+    Introduction
+
+ This source produces Chapter 1: Introduction
+ https://w3c.github.io/html/introduction.html
+
+  All the content is "non-normative". It covers:
+  - a background / potted history of HTML
+  - a few design notes
+  - a note on HTML and XML syntax
+  - information on how the spec is written and how to read it
+  - some privacy considerations
+  - an introduction to HTML itself, how to write it, etc.
+  - non-normative conformance requirements for content
+  - links to some other specs: charmod, unicode security, and some accessibility specs
```

Comment:
Unicode - is a proper noun so should start with a capital letter. I think.

Reply:
updated